pkg/kritis/metadata/metadata.go

```diff
@@ -59,13 +67,36 @@ type Vulnerability struct {
 	CVE             string
 }
 
+// RawAttestation is an internal wrapper for PgpAttestations and
+// GenericAttestations. It contains signatures, which are verified by
+// public keys held by the AttestationAuthority.
+type RawAttestation struct {
+	AttestationType    AttestationType
+	PGPAttestation     PGPAttestation
+	GenericAttestation GenericAttestation
+}
+
 // PGPAttestation represents the Signature and the Signer Key Id from the
 // containeranalysis Occurrence_Attestation instance.
 type PGPAttestation struct {
-	Signature string
-	KeyID     string
 	// OccID is the occurrence ID for containeranalysis Occurrence_Attestation instance
-	OccID string
+	OccID     string
```

Comment:
Occurrence ID should be irrelevant for RawAttestation or for PgpAttestation: when you have a RawAttestation you should no longer care where it came from, is it from Grafeas or delivered by mail :)

I understand that it is used in integration/attestation.go, however that seems like a "smell" (leaky abstraction).  Let's think about how to fix this.

Reply:
Hey, look, I think that we can just delete integration/attestation.go, and then remove Occurrence ID from here.

Reply:
Sounds good! One question, after we delete OccId, RawAttestation will just contain a Signature interface. RawAttestation seems like an unnecessary wrapper in that case. Can we remove this and just use Signature's?

Reply:
+1, and also echoing on my comment on RawAttestation, I think one point of that struct is to get rid of anything Grafeas-y.

Reply:
Done.